### PR TITLE
Feature/user profile

### PR DESCRIPTION
**Pull request by blueQcloud**

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
The committed `app/.watchman-cookie-helen-UX430UA-19066-9` file should be deleted

**Comment by wenzowski** [comment]
Looks like the schema is not in sync because you don't have https://github.com/bcgov/cas-ggircs deployed to your local database. I believe @matthieu-foucault is working on removing this requirement, but for now you should `sqitch deploy` the cas-ggircs project before you `sqitch deploy` the schema here.

**Comment by wenzowski** [comment]
```
FAIL  tests/containers/Industry/IndustryUser.test.js
  ● Test suite failed to run

    Your test suite must contain at least one test.
```
https://circleci.com/gh/bcgov/cas-ciip-portal/2421?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by wenzowski** [comment]
@blueQcloud please do make sure you `yarn test` before committing ;) this is [still failing](https://circleci.com/gh/bcgov/cas-ciip-portal/2451) because you've got that one empty test file
```
$ jest
 FAIL  tests/containers/Industry/IndustryUser.test.js
  ● Test suite failed to run

    Your test suite must contain at least one test.
```

**Comment by wenzowski** [comment]
You can always check the [current state of develop](https://circleci.com/gh/bcgov/workflows/cas-ciip-portal/tree/develop) before rebasing; every pull request should have passing tests before we merge.

File: app/containers/Industry/InputHook.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {useState} from 'react';
+
+export const useInput = initialValue => {
```
Comment (matthieu-foucault) [inline app/containers/Industry/InputHook.js]: I am not sure what this file is supposed to do. Can you add come comments please?

File: app/containers/Industry/UserDetail.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+            Submit
+          </Button>
+        </Form>
+      );
+    }
+
+    return <div />;
+  };
+
+  console.log(props);
```
Comment (matthieu-foucault) [inline app/containers/Industry/UserDetail.js]:
```suggestion
```

File: app/containers/Industry/UserDetail.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  width: '75px'
+};
+const table = {
+  marginTop: '30px',
+  marginBottom: '50px'
+};
+
+export const UserDetail = props => {
+  const {user} = props;
+  const [editMode, setMode] = useState(0);
```
Comment (matthieu-foucault) [inline app/containers/Industry/UserDetail.js]: Why is this a number and not a boolean?

File: app/containers/Industry/InputHook.js (hunk 1 above)
Comment (blueQcloud) [reply app/containers/Industry/InputHook.js]: sure, it's just a custom hook to handle form input

File: app/containers/Industry/UserDetail.js (hunk 3 above)
Comment (blueQcloud) [reply app/containers/Industry/UserDetail.js]: i used a number bc its easier to update inline for onclick

Comment (matthieu-foucault) [reply app/containers/Industry/UserDetail.js]: `setMode(true)` or `setMode(!editMode)` are easy with a boolean.

**Pull request merged**